### PR TITLE
Quickinstall

### PR DESCRIPTION
**Pull request by alstory**

### CODE REVIEW

**Comment by keitwb** [comment]
Make the new quickstart a new file under `docs/` (or even better, just edit the existing `docs/smart-agent-quickstart.md` file so we don't have two different quick starts).  Leave the original README.md as it is, we can change which file gets synced to the integrations repo easily.

**Comment by alstory** [comment]
Ben - the quickstart is different from the Quick Install, which is a
response to Joe's feeling that the installation instructions are too
complicated now. He wanted me to streamline them to make more of an "easy
button" approach. The simplest path that works for most installs was what
he was looking for. Also, there seem to be several versions of the
installation instructions that he wants to eliminate once the installation
instructions are simplified and verified correct.
I think Joe was looking for you to comment on how to Confirm the
Installation -- what would be good there for user feedback? And he is also
looking for some Troubleshooting info from you.
If you take another look at that page, you will see "stubs" for those
topics. That is where Joe was hoping you would comment.

File: SmartAgentConcepts.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# SignalFx Smart Agent Concepts
```
Comment (keitwb) [inline SmartAgentConcepts.md]: Put this file at `docs/concepts.md`.  Docs should all go under `docs/`, except for the main README.md at the root.

File: OriginalREADME.md [hunk 2]
```diff
@@ -0,0 +1,379 @@
+# SignalFx Smart Agent
```
Comment (keitwb) [inline OriginalREADME.md]: Remove this file if the content is now in other documents.

File: OriginalREADME.md [hunk 3]
```diff
@@ -0,0 +1,379 @@
...
+trials, but for full-scale deployments you will probably want to use a
+configuration management system like Chef or Puppet.
+
+##### Linux
+You can [view the source for the installer script](./deployments/installer/install.sh)
+and use it on your hosts by running:
+
+```sh
+curl -sSL https://dl.signalfx.com/signalfx-agent.sh > /tmp/signalfx-agent.sh
+sudo sh /tmp/signalfx-agent.sh YOUR_SIGNALFX_API_TOKEN
```
Comment (mstumpfx) [inline OriginalREADME.md]:
```suggestion
sudo sh /tmp/signalfx-agent.sh YOUR_SIGNALFX_API_TOKEN --realm YOUR_SIGNALFX_REALM YOUR_SIGNALFX_API_TOKEN
```

File: OriginalREADME.md [hunk 4]
```diff
@@ -0,0 +1,379 @@
...
+- [.Net Framework 3.5](https://docs.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows-10) (Windows 8+)
+- [Visual C++ Compiler for Python 2.7](https://www.microsoft.com/EN-US/DOWNLOAD/DETAILS.ASPX?ID=44266)
+
+The installer script is written for Powershell v3.0 and above and will not function correctly on earlier versions.
+
+Once the dependencies have been installed, please run the installer script below.
+You can [view the source for the installer script](./deployments/installer/install.ps1)
+and use it on your hosts in powershell by running:
+
+`& {Set-ExecutionPolicy Bypass -Scope Process -Force; $script = ((New-Object System.Net.WebClient).DownloadString('https://dl.signalfx.com/signalfx-agent.ps1')); $params = @{access_token = "YOUR_SIGNALFX_API_TOKEN"}; Invoke-Command -ScriptBlock ([scriptblock]::Create(". {$script} $(&{$args} @params)"))}`
```
Comment (mstumpfx) [inline OriginalREADME.md]:
```suggestion
`& {Set-ExecutionPolicy Bypass -Scope Process -Force; $script = ((New-Object System.Net.WebClient).DownloadString('https://dl.signalfx.com/signalfx-agent.ps1')); $params = @{access_token = "YOUR_SIGNALFX_API_TOKEN"; ingest_url = "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com"; api_url = "https://api.YOUR_SIGNALFX_REALM.signalfx.com"}; Invoke-Command -ScriptBlock ([scriptblock]::Create(". {$script} $(&{$args} @params)"))}`
```

File: README.md [hunk 5]
```diff
@@ -1,189 +1,95 @@
...
-[configured](./docs/config-schema.md#writer) in the writer, but this is generally
-only necessary if you have a very large number of metrics flowing through a
-single agent.
+ - [Confirmation](#confirmation)
+ - [Troubleshooting](#troubleshooting-the-installation)
+ - [Next Steps](#next-steps)
+ - [Other methods of Installation](#other-methods-of-installation)
+ 
 
 ## Installation
```
Comment (barbara-sfx) [inline README.md]: I think this section needs a bit of intro, along the lines of - The first time you install the SignalFx Smart Agent, we recommend you install on a single host following the instructions in this section. However, if you prefer, you can use a _Linux tar file_ (link) or _Windows zip file_ (link).

File: README.md [hunk 6]
```diff
@@ -232,7 +149,7 @@ the output to a log file or other log management system.  See the
 [signalfx-agent command](./docs/signalfx-agent.1.man) doc for more information on
 supported command flags.
```
Comment (barbara-sfx) [inline README.md]: Should they now continue with the Confirmation step? If so, add something like To confirm you are receiving data, see _Confirmation_ (link). 

Same applies to the Windows zip instructions.

File: OriginalREADME.md [hunk 7]
```diff
@@ -0,0 +1,379 @@
+# SignalFx Smart Agent 
+
+[![GoDoc](https://godoc.org/github.com/signalfx/signalfx-agent?status.svg)](https://godoc.org/github.com/signalfx/signalfx-agent)
+[![CircleCI](https://circleci.com/gh/signalfx/signalfx-agent.svg?style=shield)](https://circleci.com/gh/signalfx/signalfx-agent)
+
+The SignalFx Smart Agent is a metric agent written in Go for monitoring
```
Comment (barbara-sfx) [inline OriginalREADME.md]: Confirm that we want to keep the info about the old agent still being used internally.

File: README.md [hunk 8]
```diff
@@ -1,189 +1,95 @@
-# SignalFx Smart Agent 
+# SignalFx Smart Agent Quick Install
```
Comment (keitwb) [inline README.md]: The README on an open source project is usually geared towards developers, not users looking for a quick guide.  Make the quick install a separate document from the README (you can move some stuff out of the README if it is too repetitive).

**Comment by alstory** [comment]
Ok, will do.

File: README.md (hunk 8 above)
Comment (barbara-sfx) [reply README.md]: As I understood the request from Joe D., he specifically asked that the readme become a quick install doc. Can you guys follow up with him to agree on the plan for Anna Lisa's work? Thanks. (I don't know his github user name so can't tag him here.)

File: OriginalREADME.md (hunk 4 above)
Comment (alstory) [reply OriginalREADME.md]: Put this change in the README file

File: OriginalREADME.md (hunk 3 above)
Comment (alstory) [reply OriginalREADME.md]: put this in the README

File: README.md (hunk 6 above)
Comment (alstory) [reply README.md]: Moved the tar and zip back into the README. Put Confirmation as a step in the Quick Install.

File: README.md (hunk 8 above)
Comment (alstory) [reply README.md]: Restored the original README and linked to the information from the Quick Install page. The plan is to make the Quick Install the first topic and delete the Overview page. Leave the more technical info in the README when Ben says developers will look for it. Rewrote the Quick Start to lay out the steps.

**Comment by alstory** [comment]
will open new one

**Pull request closed without merging**